### PR TITLE
PMM-3508 Adding Support for UI tests with OVF builds

### PR DESCRIPTION
**Pull request by puneet0191**
This PR implements the following:

1. Updates the vbox-staging-start Job, so that the latest ovf image tags can be started along with setup for pmm-framework. 

2. Adds a new Job, which will be triggered as soon as a new OVF tag is created, which will then run UI tests on OVF installation of PMM on sauce labs.

### CODE REVIEW

File: pmm/vbox-staging-start.groovy [hunk 1]
```diff
@@ -119,155 +140,78 @@ pipeline {
...
+                    ssh -i "/tmp/${env.VM_NAME}/sshkey" -o ConnectTimeout=1 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null admin@${env.IP} "
+                        set -o errexit
+                        set -o xtrace
+                        sudo pmm-admin config --server ${env.IP}
+                        sudo service mysqld start
+                        sleep 10
+                        sudo service mysqld status
+                        sudo pmm-admin add mysql ms1 --host localhost
+                        sudo yum -y install wget svn cyrus-sasl-devel
+                        ls -lh /usr/lib64/libsasl*
```
Comment (hors) [inline pmm/vbox-staging-start.groovy]: Is it really needed? or was added just for debug ?

Comment (puneet0191) [reply pmm/vbox-staging-start.groovy]: yes was added for debugging, will remove it..thanks :+1:

File: pmm/pmm-ovf-testsuite.groovy [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                        sh """
+                            set +x
+                            ssh -i "/tmp/${env.VM_NAME}/sshkey" -o ConnectTimeout=1 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null admin@${env.VM_IP} "
+                                set -o errexit
+                                set -o xtrace
+                                cd pmm-qa
+                                echo $PATH
+                                node --version
+                                touch TEST.log
+                                ls -la
```
Comment (hors) [inline pmm/pmm-ovf-testsuite.groovy]: and this one

Comment (hors) [reply pmm/pmm-ovf-testsuite.groovy]: please remove it too

**Comment by puneet0191** [comment]
@hors could you please review it again, I fixed the comments you mentioned

**Pull request merged**